README.md

```diff
@@ -83,17 +84,6 @@ sudo rpm -i ~/Downloads/dunes*.rpm
 ```
 Your download path may vary.
 
-#### DUNES installation - verification
-
-Finally, verify DUNES was installed correctly in `/usr/opt/DUNES/`. First [add DUNES to Path](#add-dunes-to-path) and check:
-```bash
-dunes --version
-```
-You should see a DUNES version number. For example:
-```
-v1.0.0
-```
-
 Latest DUNES docker image will be downloaded automatically when starting the DUNES as described in [Node management](#node-management).
 #### Add DUNES to PATH
```

Comment:
This step can be deleted after we release DUNES v2.0.0.
Installation on Linux automatically adds `dunes` to $PATH.
For now I think it can stay as is.